WebApplication/appsettings.json

```diff
@@ -44,5 +44,9 @@
         "Microsoft.Hosting.Lifetime": "Information"
       }
     }
+  },
+  "Publisher": {
+    "CompletionCheck": "Callback",
+    "CallbackUrlBase": "https://inventor-config-demo.autodesk.io/complete/"
```

Comment:
@hradilm Base url should be changed for our dev. (via environment strings?  not sure how you are doing it :) )

Reply:
Does it needs to be configured? I am just curious if this cannot be generated.

Reply:
Maybe it's not... or maybe we have it already, because somehow we are generating callback url for oauth. I'll check

Reply:
@mivasi Seems we can do it, except the case when we are using `ngrok`.
So I think it worth to "calculate" callback URL if it's not specified.  I'll do it.